website/project/model.py

```diff
@@ -148,13 +147,31 @@ class MetaData(GuidStoredObject):
     date_created = fields.DateTimeField(auto_now_add=datetime.datetime.utcnow)
     date_modified = fields.DateTimeField(auto_now=datetime.datetime.utcnow)
 
+class Commentable(object):
```

Comment:
We're trying to prevent `website/project/model.py` from growing much bigger than it already is, so how about moving this into a separate module, e.g. `website.project.commentable`.

Eventually we'll want to move the Comment model outside of this module as well, e.g. `website.comments.model.Comment` (and then we could move `Commentable` to `website.comments.commentable.Commentable`. But that can wait; you've probably had enough refactoring for this PR =)